src/ape_console/_cli.py

```diff
@@ -24,6 +26,24 @@ def cli(cli_ctx, network):
     return console(verbose=verbose)
 
 
+def load_consolerc(namespace):
```

Comment:
Can we add at least a return type here?

Reply:
Yeah, sorry, I didn't see type hints elsewhere in the file but happy to add them!